simulationworkflowschema/max_ent.py

```diff
@@ -17,93 +17,58 @@
 #
 from nomad.metainfo import SubSection, Quantity, Reference
 from nomad.datamodel.metainfo.simulation.method import Method, DMFT as DMFTMethodology
-from nomad.datamodel.metainfo.simulation.calculation import BandGap, Dos, GreensFunctions
-from .general import SimulationWorkflowResults, SimulationWorkflowMethod, SerialSimulation
+from .general import (
+    DMFTOutputs,
+    MaxEntOutputs,
+    SimulationWorkflowMethod,
+    BeyondDFT2Tasks,
+)
 
 
-class MaxEntResults(SimulationWorkflowResults):
-    '''Groups DMFT and MaxEnt outputs: greens functions (DMFT, MaxEnt), band gaps (MaxEnt),
+class MaxEntResults(DMFTOutputs, MaxEntOutputs):
```

Comment:
why not simpy put here the quantities under MaxEntOutputs, are there future quantities that cannot be put in MaxEntOutputs?

Reply:
See comment before.

I wanted to define the single task outputs which then will be stored in the workflow, and keep it separated from the general Results workflow section (which also contains outputs from the other tasks)